baseplate/core.py

```diff
@@ -190,6 +190,19 @@ def configure_tracing(self, tracing_client, *args, **kwargs):
 
         self.register(TraceBaseplateObserver(tracing_client))
 
+    def configure_crash_reporting(self, client):  # pragma: nocover
```

Comment:
would `error` be a better word instead of `crash` here and throughout? `crash` seems a bit off of the main language used for this kind of thing ("error", "exception").